src/golang/cmd/migrator/versions/000013_add_workflow_dag_engine_config/up_postgres.go

```diff
@@ -0,0 +1,6 @@
+package _000013_add_workflow_dag_engine_config
+
+const upPostgresScript = `
+ALTER TABLE workflow_dag
+ADD COLUMN engine_config JSONB NOT NULL
+DEFAULT '{"type":"aqueduct", "aqueduct_config":{}}'::jsonb;`
```

Comment:
what will happen to existing rows? Will they be back-filled with this given value?

Reply:
Yes, by adding a `DEFAULT` all existing rows are given that value for the new column. If we didn't have a `DEFAULT` (and did not have `NOT NULL`) then the values would be `NULL`.